aws/index.js

```diff
@@ -0,0 +1,12 @@
+import aws from "aws-sdk";
+import nodemailer from "nodemailer";
+
+// configure AWS SDK
+const ses = new aws.SES({
+    apiVersion: "2010-12-01",
+});
+
+// create Nodemailer SES transporter
+export const transporter = nodemailer.createTransport({
+    SES: { ses, aws },
```

Comment:
Is it required to pass in `aws` as well?

Reply:
Will look into this and update accordingly

Reply:
Yes, it's required, just confirmed by testing it without the `aws` field

Reply:
https://github.com/uwblueprint/dancefest-web/blob/staging/aws/index.js I remember earlier this year when I set it up, I didn't need to. Just curious, what error are you getting when you don't specify `aws`?

Reply:
<img width="937" alt="Screen Shot 2021-06-14 at 4 40 53 PM" src="https://user-images.githubusercontent.com/32875500/121956608-577e8b00-cd2f-11eb-878b-a13042771d74.png">

I get this error when calling without the `aws` field. Maybe Nodemailer updates? I followed their documentation to implement this and they had `aws` https://nodemailer.com/transports/ses/